gtk/gtk.list-box.lisp

```diff
@@ -0,0 +1,585 @@
+;;;     gtk_list_box_selected_foreach
+;;;     gtk_list_box_get_selected_rows
+;;;     gtk_list_box_set_selection_mode                    -> Accessor
+;;;     gtk_list_box_get_selection_mode                    -> Accessor
+;;;     gtk_list_box_set_activate_on_single_click          -> Accessor
+;;;     gtk_list_box_get_activate_on_single_click          -> Accessor
+;;;     gtk_list_box_get_adjustment
+;;;     gtk_list_box_set_adjustment
+;;;     gtk_list_box_set_placeholder
+;;;     gtk_list_box_get_row_at_index
+;;;     gtk_list_box_get_row_at_y
+;;;     gtk_list_box_invalidate_filter
+;;;     gtk_list_box_invalidate_headers
+;;;     gtk_list_box_invalidate_sort
+;;;     gtk_list_box_set_filter_func
+;;;     gtk_list_box_set_header_func
+;;;     gtk_list_box_set_sort_func
+;;;     gtk_list_box_drag_highlight_row
+;;;     gtk_list_box_drag_unhighlight_row
+;;;     gtk_list_box_bind_model
+;;;     gtk_list_box_row_new
+;;;     gtk_list_box_row_changed
+;;;     gtk_list_box_row_is_selected
+;;;     gtk_list_box_row_get_header
+;;;     gtk_list_box_row_set_header
+;;;     gtk_list_box_row_get_index
+;;;     gtk_list_box_row_set_activatable                   -> Accessor
+;;;     gtk_list_box_row_get_activatable                   -> Accessor
+;;;     gtk_list_box_row_set_selectable                    -> Accessor
+;;;     gtk_list_box_row_get_selectable                    -> Accessor
+;;;
+;;; Object Hierarchy
+;;;
+;;;     GObject
+;;;     ╰── GInitiallyUnowned
+;;;         ╰── GtkWidget
+;;;             ╰── GtkContainer
+;;;                 ├── GtkBin
+;;;                 │   ╰── GtkListBoxRow
+;;;                 ╰── GtkListBox
+;;;
+;;; Implemented Interfaces
+;;;
+;;; GtkListBox implements AtkImplementorIface and GtkBuildable.
+;;;
+;;; GtkListBoxRow implements AtkImplementorIface, GtkBuildable and
+;;; GtkActionable.
+;;; ----------------------------------------------------------------------------
+
+(in-package :gtk)
+
+;;; ----------------------------------------------------------------------------
+;;; struct GtkListBox
+;;; ----------------------------------------------------------------------------
+
+#+gtk-3-10
+(define-g-object-class "GtkListBox" gtk-list-box
+  (:superclass gtk-container
+   :export t
+   :interfaces ("AtkImplementorIface"
+                "GtkBuildable")
+   :type-initializer "gtk_list_box_get_type")
+  ((activate-on-single-click
+    gtk-list-box-activate-on-single-click
+    "activate-on-single-click" "gboolean" t t)
+   (selection-mode
+    gtk-list-box-selection-mode
+    "selection-mode" "GtkSelectionMode" t t)))
+
+;;; ----------------------------------------------------------------------------
+;;; struct GtkListBoxRow
+;;; ----------------------------------------------------------------------------
+
+#+gtk-3-10
+(define-g-object-class "GtkListBoxRow" gtk-list-box-row
+  (:superclass gtk-bin
+   :export t
+   :interfaces ("AtkImplementorIface"
+                "GtkBuildable"
+                ;; "GtkActionable"
```

Comment:
Why is GtkActionable commented out?

Reply:
Documentation says it implements this interface, but that seems to be a mistake. I haven't checked yet if that interface was removed at some point, but that seems weird too?